### PR TITLE
Update node interface

### PR DESCRIPTION
**Pull request by junha1**

### CODE REVIEW

File: node/src/lib.rs [hunk 1]
```diff
@@ -112,3 +113,14 @@ pub trait SimperbyApi {
...
 }
+
+/// A working Simperby node.
+pub type SimperbyNode = node::Node<
+    simperby_network::primitives::DummyGossipNetwork,
+    simperby_network::storage::StorageImpl,
+    simperby_repository::raw::RawRepositoryImpl,
+>;
+
+pub async fn initiliaze(_config: Config) -> Result<SimperbyNode> {
```
Comment (TomTaehoonKim) [inline node/src/lib.rs:124]:
```suggestion
pub async fn initialize(_config: Config) -> Result<SimperbyNode> {
```
..? :(

File: repository/src/lib.rs [hunk 2]
```diff
@@ -32,6 +32,15 @@ impl fmt::Display for CommitHash {
 
 pub type Error = anyhow::Error;
 
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct Config {
+    /// Public repos (usually mirrors) for the read-only accesses
+    ///
+    /// They're added as a remote repo, named `public_#`.
+    /// Note that they are not part of `known_peers`.
```
Comment (TomTaehoonKim) [inline repository/src/lib.rs:40]:
```suggestion
    /// Note that they are not part of the `known_peers`.
```

File: node/src/lib.rs [hunk 3]
```diff
@@ -26,6 +27,11 @@ pub struct Config {
 
     pub broadcast_interval_ms: Option<u64>,
     pub fetch_interval_ms: Option<u64>,
+
+    /// Public repos (usually mirrors) for the read-only accesses
+    ///
+    /// They're added as a remote repo, named `public_#`.
```
[the remark this answers is not in the document]
Comment (junha1) [reply node/src/lib.rs:33]: element index

**Review by TomTaehoonKim — APPROVED** [review]
LGTM

**Pull request merged**